tests/test_logging.py

```diff
@@ -11,6 +11,7 @@
     getLogger
 )
 
+setOverrideLogLevel('INFO')
```

Comment:
Same here. Why do an override already? That way you cannot test overriding inside the function.

Reply:
Oops, your right! That's from my first shot to please the failing tests! Which turned out to be completely useless! 
I'll drop both asap!